### ISSUE TITLE
How to make OS X compatible?

### ISSUE THREAD

**Issue by MrMormon** [createuniverses/praxis#6]
Do you know what needs to be done to support Mac OS X? qmake seems to assume non-Windows equals alsa drivers!

**Comment by createuniverses** [comment]
For non Win32 builds, audio is provided by SDL. I don't know how SDL audio is implemented for Mac OS X. Are you able to compile and run a different program that uses SDL for its audio for your Mac?

**Comment by MrMormon** [comment]
Yes, and it probably used PortAudio which is available on Linux and Windows too.

Unrelated to this 'issue', are you aware of [Terra](https://github.com/zdevito/terra)? It's like a compiled gc-free Lua with interoperability with the Lua metacompile/preprocessor. It looks like a better choice for me than Julia, Rust, or Haxe for wrapping a raytracer I want to use in a game engine (idea) of mine.
